corehq/motech/openmrs/finders.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import division
+from collections import namedtuple
+
+from jsonpath_rw import parse
+
+
+PATIENT_FINDERS = []
+
+
+def register_patient_finder(class_):
+    PATIENT_FINDERS.append(class_)
+    return class_
```

Comment:
nit: is this return required because this a decorator and should return the `class_`?

Reply:
Yes, exactly.

```python
@foo
class Bar:
    pass
```
is [the same](https://www.python.org/dev/peps/pep-3129/#id11) as
```python
class Bar:
    pass
Bar = foo(Bar)
```